.husky/pre-commit

```diff
@@ -1,4 +1,8 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
+# ESLint via Turborepo
+# npm run lint -- -- --fix
```

Comment:
I think we want to enable this before the merge?

Reply:
@LekoArts We'll likely want to do this in a follow-on PR as it's going to want to update a number of files.